universe.py

```diff
@@ -69,11 +62,10 @@ def getFuture(self, dtList, selectedObj=None, dtStepSize=1e2):  # in seconds
             foundSelected = False
             mlosNew = []
             for obj in mlos:
-                if obj.x == selectedObj.x and obj.y == selectedObj.y:
-                    if obj.vx == selectedObj.vx and obj.vy == selectedObj.vy:
-                        selectedObj = obj
-                        foundSelected = True
-                        continue
+                if obj.kinematics == selectedObj.kinematics:
```

Comment:
This is where the bug was introduced. Using a variation on the old code, it works.

Reply:
Fixed in 7319782